tasks/unit-tests/junit_tests.py

```diff
@@ -146,6 +142,6 @@ def test_e2e(self, mock_popen, mock_gitlab):
         mock_project = MagicMock()
         mock_project.pipelines.get.return_value = MagicMock()
         mock_gitlab.return_value = mock_project
-        junit.junit_upload_from_tgz("tasks/unit-tests/testdata/junit-tests_deb-x64-py3.tgz")
+        junit.junit_upload_from_tgz("tasks/unit-tests/testdata/testjunit-tests_deb-x64-py3.tgz")
```

Comment:
Could you confirm that this in only cosmetic and not strictly related with this PR?

Reply:
This is related to the PR (second item in the PR description). As the name of the test file was similar to a real test there was the risk to hit the test file in case of wildcard usage in the junit file name to handle. It occurred with windows so I'm changing this name to be sure it will never happen